File: parserBrent.py
# coding=UTF-8
import sys

filename = str(sys.argv[1])

#print filename

#filename = raw_input('檔名：')
 
f = open(filename, 'r')
b_str = f.read()
f.close()
 
#print b_str.decode('utf-8') # 這是什麼？
#print b_str.decode('utf-8').encode('utf-8') # 這是什麼？


index = b_str.find("昨收")
index = index + 36
date = b_str[index:]
date = date[:8]
print 'date=' + date
print 'type=cnyes_Brent'

index = index + 27
price = b_str[index:]
priceIndex = price.find('</td><td class="')
price = price[:priceIndex]
print 'price=' + price

File: parserBrent.pl
my $s;

open(HTM, '20140419Brent') or die "$!";
while (defined($line = <HTM>)) {
	$s = $s.$line;
}
close(HTM);

print "\n========================\n";


my $last = "昨收";
my $range = index($s, $last);
#print ($s);
print ($range);

print "\n\n";

#print substr($s, $range + 30, 8);

#my $date = substr($s, $range + 30, 8);

my $date = substr($s, $range + 36, 8);

#print "\n\n";

#print substr($s, $range + 57, 6);

my $price = substr($s, $range + 63, 6);

my $tar = "Brent ".$date." ".$price;

print $tar;
print "\n========================\n";
print "\n\n";
